### PR TITLE
Highfrequency blade + Witchhunter hat changes

### PR DESCRIPTION
**Pull request by bunny232**
## About The Pull Request
Witchhunter hat no longer obscures mask/ears/face/hair.
High frequency blade is now based off the claymore instead of the scythe (the holy items). It also has a (meh) belt sprite!
This means it has a 30 block chance but looses the sycthes armor pen of 35

## Why It's Good For The Game
Witchhunter hat now has slightly more realistic properties, and so i can see the cute fox ears while wearing it. It also doesn't make gas masks/masks invisible (wacky).
Changing around a extremely underused item to a slightly more usable version is nice, also makes me happy.
Don't really think people have a reason to harp on these changes

## Changelog
:cl:
tweak: Witchhunter hat no longer obscures mask/ears/face/hair
tweak: High frequency blade is now based off the claymore instead of the scythe (the holy items), Mechanical changes: it can fit on your belt, has a block chance of 30 instead a armor pen of 35.
/:cl:

### CODE REVIEW

**Comment by YakumoChen** [comment]
Ehhhhhh... I think the sword is more thematic with the scythe's stats, not the claymore. you do know what a high frequency blade is right? It vibrates the edge as to cut better (like you would see in electric carvers or bread slicing machines, but on a better scale). It's supposed to be sharp, not hard. "All swords are good for blocking" isn't the best conception either.

Something tells me you just use the blade a lot yourself and want better gamer stats on it.

File: code/game/objects/items/holy_weapons.dm [hunk 1]
```diff
@@ -168,6 +168,7 @@
 	icon_state = "witchhunterhat"
 	item_state = "witchhunterhat"
 	flags_cover = HEADCOVERSEYES
+	flags_inv = HIDEEYES|HIDEHAIR
```
Comment (Linzolle) [inline code/game/objects/items/holy_weapons.dm:171]: this... isn't even what you wrote in the changelog. it still has the HIDEHAIR flag. Did you mean only HIDEEYES?

**Comment by Linzolle** [comment]
> 
> 
> Ehhhhhh... I think the sword is more thematic with the scythe's stats, not the claymore. you do know what a high frequency blade is right? It vibrates the edge as to cut better (like you would see in electric carvers or bread slicing machines, but on a better scale). It's supposed to be sharp, not hard. "All swords are good for blocking" isn't the best conception either.
> 
> Something tells me you just use the blade a lot yourself and want better gamer stats on it.

well technically Raiden _does_ use the HF-Blade to block in MGR but armour penetration does make more sense. Raiden also wears the sword on his back instead of his hip.

**Comment by bunny232** [comment]
making pr's while sleep deprived isnt the best of ideas

**Comment by bunny232** [comment]
just going to do the witchhat thing

**Pull request closed without merging**